rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/TableMeta.java

```diff
@@ -39,11 +39,11 @@
      * key: column name
      */
 
-    private Map<String, ColumnMeta> allColumns = new LinkedHashMap<>();
+    private Map<String, ColumnMeta> allColumns = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
```

Comment:
What s the purpose of LinkedHashMap change to TreeMap?

Reply:
Map#get & Map#containsKey of TreeMap will ignore case